.travis.yml

```diff
@@ -16,7 +16,7 @@ before_install:
 script:
   - golangci-lint run
   - go test -v --timeout 30s ./... && (CGO_ENABLED=0 GOOS=linux go build -ldflags '-d')
-  - if [ "$TRAVIS_BRANCH" == "master" ] && [ "${TRAVIS_GO_VERSION::3}" == "${PRODUCTION_GO_VERSION}" ]; then
```

Comment:
Generally good fix, but swapping single `[` for `[[` means you're executing the `test` command instead of using the bash built-in (Travis uses Bash). No reason to do that. https://unix.stackexchange.com/questions/306111/what-is-the-difference-between-the-bash-operators-vs-vs-vs

Reply:
@relistan Was it not the other way around? I thought `[` resolves to the `test` command and `[[` is a builtin:
```
> which [
/usr/bin/[
```

The `[ ... ]` version doesn't support the `"[ ${TRAVIS_GO_VERSION}" = "${PRODUCTION_GO_VERSION}"* ]` incantation, so I'd have to use something like this:
```
> export TRAVIS_GO_VERSION="1.12.x"
> export PRODUCTION_GO_VERSION="1.12"
> if [ "${TRAVIS_GO_VERSION}" != "${TRAVIS_GO_VERSION#${PRODUCTION_GO_VERSION}}" ]; then echo yes; fi
yes
> export PRODUCTION_GO_VERSION="1.11"
> if [ "${TRAVIS_GO_VERSION}" != "${TRAVIS_GO_VERSION#${PRODUCTION_GO_VERSION}}" ]; then echo yes; fi
```

Should I change to the latter?

Reply:
Sorry, that's what I get for trying to read diffs on my phone. Looked like it was changed the opposite way :( . My bad. Looks fine.